test/controllers/income/bbsi/BbsiCloseAccountControllerSpec.scala

```diff
@@ -189,7 +188,7 @@ class BbsiCloseAccountControllerSpec extends PlaySpec
 
         val sut = createSUT
 
-        val date = TaxYearResolver.endOfCurrentTaxYear.plusDays(1)
+        val date = TaxYear().next.start
```

Comment:
Is this the same as TaxYear().end?

Reply:
TaxYearResolver.endOfCurrentTaxYear is the same as TaxYear().end